source/errors.rst

```diff
@@ -174,8 +176,6 @@ server is running.
     "text": "Invalid username or password" / "Bad oAuth token" / etc
```

Comment:
Do you want to do the same thing here you did with server error?  I think it would work well.